### DISCUSSION TITLE
Does Postal includes a mail server?

### DISCUSSION THREAD

**Discussion by parasiteoflife** [postalserver/postal#3144] [Help with using Postal]
- I'm already hosting my mail, right now I only need an HTTP API to send mails, can I use Postal only for this?
- Does Postal includes a full mail server?
- I understand that Postal can send mass marketing mails like Mailgun so if I already have a mail server (which is using all the corresponding ports 25, 587, etc) can I still use this?

Thank you!

**Comment by willpower232** [comment]
Postal can send email it is asked to over HTTP or SMTP, regardless of the port it is sending to. 

I would say it isn't really a full mail server, it doesn't have inboxes or most of the features in a traditional sense.

It only supports explicit SSL with STARTTLS so as long as you don't expect to use other ports than 443 or 25 out of the box to send your message to Postal, then you can probably use it.

I would also point out that Postal does not have the marketing features like templates or mailing lists or unsubscribes so if you're handling them elsewhere then you're probably sorted.

**Reply by parasiteoflife** [reply]
If I'm only going to use the HTTP API, do I still need to use port 25 with Postal? That port is already used by the mail server

**Reply by willpower232** [reply]
You may be able to get away with it however you definitely wouldn't be able to get bounces or use the incoming mail features in Postal so it could be a bit confusing.
